lib/install.js

```diff
@@ -6,6 +6,11 @@ var _ = require('lodash');
 var assert = require('assert-plus');
 var vasync = require('vasync');
 var verror = require('verror');
+var compose = require('restify').helpers.compose;
+var bunyan = require('bunyan');
+var LOG = bunyan.createLogger({name: 'enroute'});
```

Comment:
Rather than create a new logger we should have enroute here create a child logger from an incoming logger - we'll make sure NQ threads it through:

```js
var log = opts.log.child({ component: 'enroute' });
```

Reply:
Doesn't that require another Restify release first, because today it doesn't pass a logger?